modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step07_extra_element_ids.md

```diff
@@ -0,0 +1,159 @@
+# Reporting IDs: A Powerful Feature for Assisting with Physics Input and Output Processing
```

Comment:
Can we get a warning here somewhere about the restriction of ExodusII-stored IDs to whatever can be safely converted from unsigned int to double and back?  We only get like 15 digits preserved, where a user might naturally expect 19 from a 64-bit uint, or might expect to be able to set negative ids.